package.json

```diff
@@ -20,13 +20,13 @@
     "yeoman-generator": "^1.1.1"
   },
   "devDependencies": {
-    "babel-eslint": "^7.2.1",
+    "babel-eslint": "^7.2.3",
     "eslint": "^4.4.1",
     "eslint-config-callstack-io": "^0.4.1",
-    "husky": "^0.13.3",
-    "jest": "^19.0.2",
-    "lint-staged": "^3.4.0",
-    "prettier": "^0.22.0"
+    "husky": "^0.14.3",
+    "jest": "^21.0.2",
+    "lint-staged": "^4.1.3",
+    "prettier": "^1.6.1"
```

Comment:
Same here, no need to depend on `prettier`.

Reply:
I see. I thought that is a need for callstack's eslint. I will remove it then. 👍